model/ConceptPropertyValue.php

```diff
@@ -87,6 +96,15 @@ public function getUri()
         return $this->resource->getUri();
     }
 
+    public function getExVocab()
+    {
+        if ($this->isExternal()) {
+            return $this->vocab;
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method getExVocab uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951447)